tests/unit/store/traction/pacbio/runCreate/run.spec.js

```diff
@@ -38,7 +38,9 @@ const wells = {
 }
 
 const wellValues = Object.values(wells)
-const plateValues = [{ wells: wellValues }]
+const plateValues = { 1: { plate_number: '1', wells: wellValues } }
```

Comment:
Check str vs int

Reply:
@StephenHulme  
Oh I think plate_number is a String in some places, but and Integer in others. Not 100%, but just wanted to check its consistent. It might just be a string here if it is coming from JSON etc

Reply:
Hm yes, agree. 

```
x = { 1: 'a value', 2: 'another value'}
x[1] (doesn't work)
x['1'] (does work)
```

So checking all plate number keys/ references are strings 👍

Reply:
Oh ok, yes interesting.
I'm happy to take your lead. The place that was concerning me is the code [here](https://github.com/sanger/traction-ui/pull/1223/files#diff-74e29ac055ed0820f225562b264573a58a459c9bf1584e282094e843e2dc3577R117-R119)